builder.json

```diff
@@ -49,7 +48,9 @@
                 "armv6": {
                     "!packages": [],
                     "!build_steps": [
-                        "build"
+                        "build",
+                        "mv target/cmake-build/aws-crt-java/* target/cmake-build/",
```

Comment:
move the cross-compiled libs to the expected lib path to create jar. json file doesn't support comment.

Previous `"build"` only use cmake to build the shared lib.